ha/core/system_health/system_health.py

```diff
@@ -276,33 +276,32 @@ def _is_update_required(self, current_health: str, updated_health: str, event: H
             healthevent (HealthEvent): health event object.
 
         Returns:
-            bool: return true if update is needed.
+            HEALTH_EVENT_ACTIONS
         """
+        event_action: HEALTH_EVENT_ACTIONS = HEALTH_EVENT_ACTIONS.UPDATEPUBLISH.value
         if not current_health:
```

Comment:
First events will be UNKNOWN events and this if will be TRUE now. So, this will return updatepublish. Right?
So, systemhealth will be updated with status as UNKNOWN and alert will be sent to HARE. Correct?

Reply:
Yes, but I guess HARE does not subscribe to UNKNOWN events for node. So, it will not reach HARE.

Reply:
Right. Thanks!